### ISSUE TITLE
Help me with item achievements.

### ISSUE THREAD

**Issue by CurioM2** [gibbed/SteamAchievementManager#308]
Hello, i'm new here and i've got some problems. Long ago, i unlocked all the achievements by this program in Team Fortress 2. In their number, there was some that will give you special items on unlock. So, recently, i deleted all achievement items and erased all the achievements in the game(there is a button that deletes them), and tried to unlock them again. But, when i unlocked them by SAM, commited my changes, and look for the game, it gave me nothing. I tried everything - guides, reset, restart. But it does nothing. I'm asking for help, if someone can give me some advices.

**Comment by Kimarnic** [comment]
No, those items are one time only, once lost you can't get them back
